dashboard/prepare-dashboard-data.py

```diff
@@ -303,10 +303,10 @@ def count_dups(hvr_fname):
         sample_metadata[sample]["reads"] = \
             project_sample_reads[project][sample]
 
-        rr_fname = "riboreads/%s.riboreads.txt.gz" % sample
+        rc_fname = "ribocounts/%s.ribocounts.txt" % sample
         try:
-            with gzip.open(rr_fname, 'rt') as file:
-                ribocount = sum(1 for _ in file)
+            with open(rc_fname, 'r') as file:
+                ribocount = file.readline()
             sample_metadata[sample]["ribocounts"] = ribocount
```

Comment:
Since this isn't a count but a fraction, how about changing the key?

Reply:
Yea I went back and forth on this. Do you mean change the key just in the dashboard metadata or accross run.py as well? Eg renaming `ribocounts()` and all mentions of ribocounts to `ribofrac`?

Reply:
I think changing all "ribocounts" to "ribofrac" would be clearest, yes. But if that feels like too much hassle just changing it in the sample metadata is better than nothing?